packages/theme/src/cli/commands/theme/push.test.ts

```diff
@@ -1,102 +0,0 @@
-import Push from './push.js'
```

Comment:
Did you mean to completely remove the push test?

Reply:
Yep, there's no more business logic in that layer anymore since we moved it to service layer for `@shopify/theme` export